lincheck/src/main/java/org/jetbrains/kotlinx/lincheck/paramgen/StringGen.kt

```diff
@@ -0,0 +1,78 @@
+package org.jetbrains.kotlinx.lincheck.paramgen
+
+import org.jetbrains.kotlinx.lincheck.paramgen.ParameterGenerator.UNIQUE_MODIFIER
+import java.lang.IllegalArgumentException
+import kotlin.random.Random
+
+/*
+ * #%L
+ * Lincheck
+ * %%
+ * Copyright (C) 2015 - 2018 Devexperts, LLC
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+
+class StringGen(configuration: String) : ParameterGenerator<String> {
+    private val genImpl: ParameterGenerator<String>
+
+    init {
+        val args = configuration.replace("\\s".toRegex(), "").split(":".toRegex()).filter { it.isNotEmpty() }
```

Comment:
it seems that this parsing function is common for all parameter generators, let's share it